extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/config/DialectVersions.java

```diff
@@ -16,10 +16,13 @@
  */
 public final class DialectVersions {
     // The following constants must be at least equal to the default dialect version in Hibernate ORM
+    public static final String DB2 = "10.5";
+    public static final String DERBY = "10.14";
     public static final String MARIADB = "10.6";
     public static final String MSSQL = "13"; // 2016
     public static final String MYSQL = "8";
     public static final String ORACLE = "12";
+    public static final String POSTGRESQL = "10.0";
```

Comment:
I think I would rather defer to ORM than enforcing all versions but that's just my 2 cents.

Reply:
Yeah, I was on the fence. It seems nobody has a strong opinion though... hard call :)